### ISSUE TITLE
Consider making WhoIsTyping an aria alert

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#11278]
As it is time-sensitive, it is currently picked up by the live-region but alert should be more semantically correct

**Comment by t3chguy** [comment]
Aria polite from aria-live is sufficient.

![image](https://user-images.githubusercontent.com/2403652/234269111-3c58e793-0cc7-4857-86a1-90eaf2f02769.png)

We have it within our polite live region already
